backend/dotnet/Quickstart/ZUMOAPPNAMEService/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Http;
+using Microsoft.Azure.Mobile.Server.Config;
+
+namespace ZUMOAPPNAMEService.Controllers
+{
```

Comment:
We may want to add some comment here describing with the [MobileAppController] attribute does -- something like "Use the MobileAppControllerAttribute to identify ApiControllers that you want to use from your mobile clients".